lib/src/util.dart

```diff
@@ -9,81 +9,24 @@ import 'assets/html_entities.dart';
 import 'charcode.dart';
 import 'patterns.dart';
 
-String escapeHtml(String html) =>
-    const HtmlEscape(HtmlEscapeMode.element).convert(html);
+/// One or more whitespace, for compressing.
+final _oneOrMoreWhitespacePattern = RegExp('[ \n\r\t]+');
+
+String escapeHtml(
+  String html, {
+  bool escapeApos = true,
+  bool escapeLtGt = true,
+  bool escapeQuot = true,
+}) =>
+    HtmlEscape(HtmlEscapeMode(
```

Comment:
I think both `HtmlEscape` and `HtmlEscapeMode` can be made `const`.

Reply:
Thanks, fixed.